packages/fhir-health-care-service/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@opensrp/fhir-health-care-service",
+  "version": "1.0.0",
+  "description": "Team management components for opensrp web",
+  "main": "dist/index.js",
+  "types": "dist/types",
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "https://github.com/OpenSRP/web/",
+  "scripts": {
+    "jest": "jest --coverage --verbose --color",
+    "test": "cd ../../ && yarn test packages/fhir-health-care-service --verbose --collectCoverage=true -u && cd packages/fhir-health-care-service",
+    "test:watch": "cd ../../ && yarn test packages/fhir-health-care-service --verbose --watch && cd packages/fhir-health-care-service",
+    "tsc": "tsc",
+    "lint": "eslint ./**/*.{js,jsx,ts,tsx}",
+    "transpile": "babel src -d dist --root-mode upward --extensions .ts,.tsx --ignore '**/*.test.ts,**/*.test.tsx,**/tests,**/__tests__'",
+    "copy": "copyfiles -u 1 \"./src/**/*.{css,html}\" \"./dist/\"",
+    "build": "yarn run tsc && yarn run transpile && yarn run copy",
+    "extractTranslations": "yarn i18next 'src/**/*.{ts,tsx}' -c ../../i18next-parser.config.js"
+  },
+  "jest": {
+    "automock": false,
+    "setupFiles": [
+      "../../setupTests"
+    ]
+  },
+  "bugs": {
+    "url": "https://github.com/OpenSRP/web/issues"
+  },
+  "author": "OpenSRP Engineering",
+  "license": "Apache-2.0",
+  "private": false,
+  "dependencies": {
+    "@opensrp/notifications": "^0.0.5",
+    "@opensrp/react-utils": "^0.0.11",
+    "uuid": "^8.3.1",
+    "@opensrp/pkg-config": "^0.0.9",
+    "react-i18next": "^11.8.10"
+  },
+  "devDependencies": {
+    "@types/antd": "^1.0.0",
+    "@opensrp/fhir-team-management": "file:../fhir-team-management",
```

Comment:
@ashfahan Should there be an issue that this needs to be updated once package is published

Reply:
https://github.com/opensrp/web/issues/879